src/Discovery/src/Eureka/Configuration/EurekaHealthOptions.cs

```diff
@@ -3,24 +3,28 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Extensions.Configuration;
+using Steeltoe.Common.HealthChecks;
 
 namespace Steeltoe.Discovery.Eureka.Configuration;
 
 public sealed class EurekaHealthOptions
 {
     /// <summary>
-    /// Gets or sets a value indicating whether to enable the management health contributor. Configuration property: eureka:client:health:enabled.
+    /// Gets or sets a value indicating whether to activate an <see cref="IHealthContributor" /> that verifies connectivity to the Eureka server. Default
+    /// value: true.
```

Comment:
```suggestion
    /// Gets or sets a value indicating whether to activate an <see cref="IHealthContributor" /> that verifies connectivity to the Eureka server.
    /// Default value: true.
```
readability nit

Reply:
Can't do this, the code cleanup formatter auto-inserts line breaks, based on the line length.
However, VS is smart enough to read through them:
![image](https://github.com/SteeltoeOSS/Steeltoe/assets/104792814/11645b35-dc8b-4004-91b1-b82f163782b3)